src/components/ChartWidgetEditor/components/DataViz/Dataviz.tsx

```diff
@@ -14,6 +10,8 @@ import getChartInput from '../../../../utils/getChartInput';
 import { CONTAINER_ID } from './constants';
 import { DashboardSettings } from '../../../../modules/dashboards';
 
+import { WidgetSettings, Tag } from '@keen.io/widgets';
```

Comment:
what's the difference between `WidgetSettings` from `@keen.io/widget-picker` and `@keen.io/widgets`?

can we unify them?

Reply:
WidgetSettings from widget-picker represents only subset of settings that could be defined in picker component. WidgetSettings from `@keen.io/widgets` contains all widget settings.